vortex-flatbuffers/flatbuffers/vortex-file/footer2.fbs

```diff
@@ -1,10 +1,19 @@
 include "vortex-layout/layout.fbs";
 
 /// A `Segment` acts as the locator for a buffer within the file.
-table Segment {
+// TODO(ngates): I think this should actually be more of a German String style struct? I don't know how
+//  many buffers we have that are <128 bits.
```

Comment:
I think this would be minor though encoding enum like strings does end up with tiny dictionaries